build.gradle

```diff
@@ -0,0 +1,110 @@
+allprojects  {
+    apply plugin: 'maven'
+
+    group = 'microprofile.gift.demo'
+    version = '0.0.1-SNAPSHOT'
+}
+
+description = 'Acme gift giving using microservices'
+
+subprojects {
+    apply plugin: 'java'
+    sourceCompatibility = 1.8
+    targetCompatibility = 1.8
+    tasks.withType(JavaCompile) {
+        options.encoding = 'UTF-8'
+    }
+
+    repositories {
+        mavenLocal()
+
+        maven { url "https://oss.sonatype.org/content/repositories/netwasdev-1087" }
+        maven { url "http://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/wasliberty-open-liberty/" }
+        maven { url "https://repo.eclipse.org/content/groups/releases/" }
```

Comment:
at this point, I am not sure what these two URLs are for - maybe it will become more obvious.  Especially the first since you are installing open liberty from maven central using `libertyRuntime`